cli/pkg/kctrl/cmd/package/repository/release/artifact_writer.go

```diff
@@ -31,11 +31,14 @@ func (w *ArtifactWriter) WritePackageRepositoryFile(imgpkgBundleLocation string)
 		ObjectMeta: metav1.ObjectMeta{
 			Name:              w.PackageRepoName,
 			CreationTimestamp: metav1.NewTime(time.Now()),
+			Annotations: map[string]string{
+				"sha-ref": bundleURLWithSHARef,
```

Comment:
2 questions:
1. Should we add this annotation? 
2. If yes, what should be the name?

Reply:
question 3, should this we flipped? (ie image uses sha reference and tag is in annotations), and if so, should we augment kctrl to display annotation value as an additional output in the cli?

Reply:
@cppforlife I thought about it. Here are my observations:
1. If users are adding the pkg repo with `kctrl pkg repo add -r ... --url ... -n ....`, or the annotation is not present, for them, kctrl will not show anything in the `list` or `get` command.
2. Only when the annotation is provided, we will add the value.
Sample output:
```
./kctrl pkg repo list -A
Target cluster 'https://192.168.64.108:8443' (nodes: minikube)

Repositories in all namespaces

Namespace   Name   Source                                                           Tag                    Status
kctrl-test  demo   (imgpkg) rohitagg2020/demo-repo:1.0.0                                  -       Reconcile succeeded
^           demo2  (imgpkg) rohitagg2020/demo-repo:sha256-8a55a777f43f6755fbe782f7b45...  -       Reconcile succeeded
^           demo3  (imgpkg) rohitagg2020/demo-repo:sha256-8a55a777f43f6755fbe782f7b45...  1.0.0   Reconcile succeeded
```

What are your thoughts?